src/index.js

```diff
@@ -161,10 +161,6 @@ checkboxes.forEach((chbox) => {
   });
 });
 
-// function editTask() { // this is function for edit task
```

Comment:
* As the previous reviewer pointed out the edit, clear all completed, completed status are not working and the local storage isn't updated with the status change 
* I get this error in the console when i click on the edit button and the check box kindly 🙏 fix it 
![image](https://user-images.githubusercontent.com/52497246/127752764-25042b2d-c008-42f6-ab35-22c9a3eef345.png)

Reply:
Dear @avidosky007,

I fixed the edit delete function, please, check.

My best regards,

Elmar

Reply:
![image](https://user-images.githubusercontent.com/49064106/127754035-5dc84b77-26c9-46c5-9a94-ff23690b70c0.png)

Reply:
As you can see on this screenshot, there is no error when checking the task in the console.

Reply:
- [ ] Dear Elmar, nice work so far. But this issue as pointed out by the previous reviewer is not resolved, kindly see the comments I leave to make the necessary changes.